src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -323,22 +324,41 @@ private void handleInvalidFileError(
             null
         );
 
-        ErrorMapping
-            .getFor(cause.getClass())
-            .ifPresent(errorCode -> {
-                try {
-                    sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
-                } catch (Exception exc) {
-                    log.error(
-                        "Error sending notification to the queue."
-                            + "File name: " + zipFilename + " "
-                            + "Container: " + containerName,
-                        exc
-                    );
-                }
-            });
+        Optionals.ifPresentOrElse(
+            ErrorMapping.getFor(cause.getClass()),
+            (errorCode) -> trySendErrorMessageAndMoveFileToRejectedContainer(
+                zipFilename,
+                containerName,
+                cause, eventId,
+                leaseId,
+                errorCode
+            ),
+            () -> {
+                throw new ConfigurationException("Error code mapping not found for " + cause.getClass().getName());
+            }
```

Comment:
unrelated

Reply:
This is what Leszek asked to change

Reply:
I asked for this after some changes were made in this PR, resulting in the lack of failure in case of misconfiguration.

All we need for this story, is not catching the exception.

Reply:
ok, I will revert this change